### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by Gracetemitope**
In this branch, we did the following:

- Create Calculator snapshot
- Operate.js unit testing
- Calculate.js unit testing

### CODE REVIEW

File: src/tests/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import calculate from "../logic/calculate";
+
```
Comment (bellom) [inline src/tests/calculate.test.js:2]: We need to write a unit test for all the functionalities & scenarios in `calculate.js` module.

Example:
Kindlly write unit test for the following:
1. `buttonName === '.'`
2. `buttonName === '='`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Gracetemitope  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

It would be nice to fix your design (calculator)
image


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/tests/calculate.test.js (hunk 1 above)
Comment (crgc) [reply src/tests/calculate.test.js:2]: We already did that. It's in a file called `calculate.test.js`. Hard to miss!

Comment (bellom) [reply src/tests/calculate.test.js:2]: Kindly double-check, as you can see, I drop a comment on `calculate.test.js`.

Comment (crgc) [reply src/tests/calculate.test.js:2]: Done.

Comment (bellom) [reply src/tests/calculate.test.js:2]: @crgc you deleted your first comment 😄, it would be nice to be professional with our comment 👍, thank you.

**Review by ZawadiSumba66 — APPROVED** [review]
### Hi Team 😃 , 
Great job implementing changes requested by the previous reviewer. 💯 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**